test/gpu/nvidia_test.go

```diff
@@ -83,6 +90,45 @@ var expectedDimsToMetrics = map[string][]string{
 	},
 }
 
+var expectedDimsToMetricsE2E = map[string][]string{
```

Comment:
Is this different than `expectedDimsToMetricsIntegTest`? why?

Reply:
The integ test does not use the current build of the agent, it uses the public image. So metrics like pod_gpu_request, pod_gpu_total, pod_gpu_limit, ... aren't begin accounted for in the integ test, but they are in the E2E because we are using the build image of CWA.